module/zfs/zio.c

```diff
@@ -4833,8 +4833,9 @@ EXPORT_SYMBOL(zio_data_buf_alloc);
 EXPORT_SYMBOL(zio_buf_free);
 EXPORT_SYMBOL(zio_data_buf_free);
 
-module_param(zio_delay_max, int, 0644);
-MODULE_PARM_DESC(zio_delay_max, "Max zio millisec delay before posting event");
+module_param(zio_slow_io_ms, int, 0644);
+MODULE_PARM_DESC(zio_slow_io_ms,
+	"Max zio completion time (milliseconds) before posting a delay event");
```

Comment:
I'm assuming this also controls when we count a slow io in `zpool status`.  Maybe we should also mention the `zpool status` impact, since that may impact more uses.